assets/stories/skate-components/userAvatar.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import React from "react"
+
+import { UserAvatar } from "../../src/components/userAvatar"
+
+const meta = {
+  component: UserAvatar,
+  decorators: [
+    (Story) => (
+      <div style={{ width: "2rem", height: "2rem" }}>
+        <Story />
+      </div>
+    ),
+  ],
+} satisfies Meta<typeof UserAvatar>
+
+export default meta
+type Story = StoryObj<typeof UserAvatar>
+
+export const Default: Story = { args: { userName: "fake@test.com" } }
```

Comment:
Nit: It might be helpful to nest these (along with the upcoming `<LoggedInAs />` component) into a subfolder, rather than having them all live top-level. I called it `identity`, but I'm not overly attached to that name.


![Screenshot 2024-02-12 at 1 48 51 PM](https://github.com/mbta/skate/assets/912020/30991ef1-12bb-4a5f-ad90-d98ed867ca56)

Just thinking that the top level of ☝️ will get a bit cluttered.

Reply:
I'd be open to moving UserAvatar, though CircleButton isn't necessarily tied to the user avatar (and I think design envisions using it in other places in the future).